### ISSUE TITLE
Link events to osm-community-index entries

### ISSUE THREAD

**Issue by 1ec5** [thomersch/openstreetmap-calendar#60]
Many events are organized or sponsored by local OSM communities, but it isn’t always clear which one by looking at the event’s name, and it isn’t straightforward for an API client to filter by community. The event creation form should have a field that accepts an [osm-community-index](https://openstreetmap.community) entry to list as the organizer.

Ideally, the field would be a dropdown of all the osm-communty-index entries, but as a stopgap, it could accept ID ([example](https://github.com/osmlab/osm-community-index/blob/fc9e0360800e54afd83bd3fbc35877e1c1cea0d5/resources/north-america/united_states/OSM-SouthBay.json#L2)) and a little bit of help text could explain how to find the iD.

#36 sounds similar, but I wasn’t sure if it was talking about the community index or some other kind of community. Hopefully the proposal here would be a simple first step towards better integration with other community-facing tools.

/cc @bhousel

**Comment by thomersch** [comment]
You're right, this is related to #36. For the community feature there is a half-way prototype, but there were several people who wanted to build establish some kind of "teams" platform and I wanted to hold it back for a while to see whether any of those will crystallise. 

I feel incredible lucky that OSMCAL got so much traction so quickly, other new platforms/features might not get accepted so quickly. Therefore osm-community-index sounds like a good hook, because it already exists.

One of the ideas for the "community feature" is the ability to extend OSMCAL beyond core of OSM, e.g. open data projects could enter events without spilling into the OSM namespace. Therefore I would make the connection to osm-community-index using a level of indirection: Users of OSMCAL could join any communities and any community can have a link to the community index.

An API to be able to retrieve events by community would be possible, too. This way communities could link to their events easily.

**Comment by 1ec5** [comment]
> Therefore I would make the connection to osm-community-index using a level of indirection: Users of OSMCAL could join any communities and any community can have a link to the community index.

Interesting, I hadn’t given thought to the joining function. It seems like that would be a useful next step after collecting information about the organizing communities in the first place. In my opinion, OSMCal has momentum that we can build upon by getting it in front of everyday mappers, and osm-community-index (which is integrated into iD’s post-save dialog) would be the most effective conduit for that.

As for being project-agnostic, I don’t think an osm-community-index ID field would be exclusive of some other field for a non-OSM resource, but openstreetmap/iD#8307 would probably be more feasible with a field that unambiguously holds an osm-community-index ID.
